pinc/ProjectState.inc

```diff
@@ -28,39 +28,44 @@ class ProjectState
 
 class ProjectStates
 {
-    private static $states = [];
+    /** @var array<string, ProjectState> */
+    private static array $states = [];
 
-    public static function add_state($state)
+    public static function add_state(ProjectState $state): void
     {
         self::$states[$state->name] = $state;
     }
 
-    public static function get_states()
+    /** @return string[] */
+    public static function get_states(): array
     {
         return array_keys(self::$states);
     }
 
-    public static function get_all()
+    /** @return array<string, ProjectState> */
+    public static function get_all(): array
     {
         return self::$states;
     }
 
-    public static function get_medium_label($state)
+    public static function get_medium_label(?string $state): string
     {
         return self::$states[$state]->medium_label ?? '';
     }
 
-    public static function get_label($state)
+    public static function get_label(?string $state): string
     {
         return self::$states[$state]->label ?? '';
     }
 
-    public static function get_forum($state)
+    /** @return string|int */
+    // TODO: Switch the return to ?string but would need to inspect usage of the function.
+    public static function get_forum(?string $state)
```

Comment:
Because I'm learning about this as we go, I just wanted to verify that this function is intentionally left without a return type (referencing the comment above it).